.github/workflows/test.yml

```diff
@@ -24,5 +24,10 @@ jobs:
       - name: Re-generate auto-generated files
         run: make auto-generated
 
+      # Needed to run tests, when they are running in compilation mode
+      # (PERLANG_EXPERIMENTAL_COMPILATION=true)
+      - name: Rebuild stdlib
+        run: make stdlib
+
```

Comment:
This runs the _compilation_ of the stdlib. We could also run the tests, but they are not fully working yet (and even when they are, there will be some more work to get them working on macOS). Let's see if we merge this now already or if we fix this first.